pkg/regionpicker/cache.go

```diff
@@ -0,0 +1,119 @@
+package regionpicker
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// RegionCachePath is the $HOME/<RegionCachePath> location of the regionpicker cache
+	RegionCachePath = filepath.Join(".outreach", ".cache", "box")
+	// RegionCacheFile is the name of the regionpicker cache file
+	RegionCacheFile = "regionpicker.json"
+)
+
+// cacheEntry is information on a cloud's cache entry
+type cacheEntry struct {
```

Comment:
I think caches should use the standard cache interface we defined [here](https://github.com/getoutreach/datastores/blob/5e7ed55abb4143256326cc95c31401005b37d944/pkg/cache/cache.go#L60).   Maybe we should add a cache.File that backs this implementation onto a single file.

If needed, we can move that interface and implementation into gobox/pkg/cache (and eventually move nearly everything in datastores/pkg/cache over to gobox/pkg/cache).